pkg/controller/nodemaintenance/nodemaintenance_controller.go

```diff
@@ -153,9 +153,9 @@ type ReconcileHandler interface {
 type ReconcileNodeMaintenance struct {
 	// This client, initialized using mgr.Client() above, is a split client
 	// that reads objects from the cache and writes to the apiserver
-	client      client.Client
-	scheme      *runtime.Scheme
-	drainer     *drain.Helper
+	client	  client.Client
+	scheme	  *runtime.Scheme
+	drainer	 *drain.Helper
```

Comment:
Apologies for being unclear...
Do not change whitespace in lines that were not otherwise modified by your patch.
The idea is to not introduce additional inconsistencies in _leading_ whitespace only.

Reply:
Usually the IDE (using indent plugins) will align those. 
There is also a whitespace check in our make that will fail the build https://github.com/kubevirt/node-maintenance-operator/blob/9fe31971cf36b090fbc6eee31f687c8cb30c3fdb/Makefile#L47   
But that doesn't apply for indents.

Reply:
I converted all spaces to tabs using the enforce-tabs-or-spaces.sh scripts from pr #66. Now upon repeated reading that contradicts with your recommendation from conversation #60, I am at a loss what to do now.